aibolit/patterns/assign_null_finder/assign_null_finder.py

```diff
@@ -24,7 +24,7 @@
 
 
 class NullAssignment:
-
+    # flake8: noqa
```

Comment:
@lukyanoffpashok why ignore?

Reply:
because flake is swearing at '\s', which is just a regular expression